test/xbmcaddon.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0 (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+import json
+import os
+import polib
+import sys
+
+# FIXME: Get information from addon.xml
+ADDON_INFO = {
+    'author': 'Martijn Moreels',
```

Comment:
not sure if deliberatly written wrong :D

Reply:
It was pretty late and I planned to rewrite this anyway ;-)
As we can get all this information directly from the addon.xml anyway.

Will fix.